.github/workflows/test.yml

```diff
@@ -8,25 +8,30 @@ jobs:
     name: Test
     runs-on: ubuntu-latest
     steps:
-
-    - name: Check out code into the Go module directory
-      uses: actions/checkout@v4
-      with:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v4
+        with:
           submodules: recursive
 
-    - name: Set up Go 1.x
-      uses: actions/setup-go@v5
-      with:
-        go-version-file: 'go.mod'
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: "go.mod"
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v6
+        with:
+          version: latest
+          args: --timeout=10m
+
+      - name: prepare for testing
+        run: sudo apt-get update && sudo apt-get install -y cabextract
 
-    - name: golangci-lint
-      uses: golangci/golangci-lint-action@v6
-      with:
-        version: latest
-        args: --timeout=10m
-        
-    - name: prepare for testing
-      run: sudo apt-get update && sudo apt-get install -y cabextract
+      - name: unshallow diff-test submodules
+        run: |
+          (cd pkg/diff/testdata/fixtures/vuls-data-extracted-diff-test.git && git fetch --unshallow)
+          (cd pkg/diff/testdata/fixtures/vuls-data-extracted-diff-test-archive-1.git && git fetch --unshallow)
+          (cd pkg/diff/testdata/fixtures/vuls-data-raw-diff-test.git && git fetch --unshallow)
```

Comment:
How about using `git -C <path>` instead of cd?
https://git-scm.com/docs/git#Documentation/git.txt--Cltpathgt

Reply:
did it!